src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -19,6 +19,9 @@ namespace Microsoft.AspNet.Mvc
     /// </summary>
     public class DefaultControllerActionArgumentBinder : IControllerActionArgumentBinder
     {
+        private static readonly MethodInfo CallPropertyAddRangeOpenGenericMethod =
+            typeof(MutableObjectModelBinder).GetTypeInfo().GetDeclaredMethod(nameof(CallPropertyAddRange));
```

Comment:
But your field still refers to `MutableObjectModelBinder.CallPropertyAddRange` :laughing: